.github/workflows/package-tests.yml

```
@@ -0,0 +1,40 @@
+          
+      - name: Setup Swift
+        uses: swift-actions/setup-swift@v1
+        with:
+          swift-version: '5.9'
+          
+      - name: Install Mint
+        run: |
+          curl -L https://github.com/yonaskolb/Mint/releases/download/0.17.5/mint.zip -o mint.zip
+          unzip mint.zip
+          sudo mv mint /usr/local/bin/
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)